### PR TITLE
[Docs] Add Snowflake target store

### PR DESCRIPTION
**Pull request by jillnogold**
[ML-5656](https://iguazio.atlassian.net/browse/ML-5656)

[ML-5656]: https://iguazio.atlassian.net/browse/ML-5656?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/feature-store/sources-targets.md [hunk 1]
```diff
@@ -199,6 +199,26 @@ For example:
 Disable partitioning with:
 - `ParquetTarget(partitioned=False)`
 
+## Snowflake target
+
+```python
+profile = DatastoreProfileSnowflakeTarget(name="profile-name",bootstrap_servers="localhost", topic="topic_name")
```
Comment (tomerm-iguazio) [inline docs/feature-store/sources-targets.md]: snowflake does not have datastore profile:)

File: docs/feature-store/sources-targets.md [hunk 2]
```diff
@@ -199,6 +199,26 @@ For example:
...
+target = SnowflakeTarget(path="ds://profile-name")
+```
+
+`DatastoreProfileKafkaTarget` class parameters:
```
Comment (tomerm-iguazio) [inline docs/feature-store/sources-targets.md]: do you mean `SnowflakeTarget` ?

**Review by tomerm-iguazio — COMMENTED** [review]
Nice! just few fixes...

File: docs/feature-store/sources-targets.md (hunk 1 above)
Comment (jillnogold) [reply docs/feature-store/sources-targets.md]: deleted

File: docs/feature-store/sources-targets.md (hunk 2 above)
Comment (jillnogold) [reply docs/feature-store/sources-targets.md]: @tomerm-iguazio - if there is no datastore profile - should that line be "SnowflakeTarget parameters"?

**Review by tomerm-iguazio — APPROVED** [review]

**Comment by tomerm-iguazio** [comment]
Can you update in the pr that `schema` in `SnowflakeSource` will be renamed to `db_schema` ?

**Comment by jillnogold** [comment]
@tomerm-iguazio - updated. Can you approve?

**Review by tomerm-iguazio — APPROVED** [review]
looks great, tnx!

**Pull request merged**